rust/src/core.rs

```diff
@@ -28,10 +28,7 @@ use crate::flow::Flow;
 pub enum DetectEngineState {}
 pub enum AppLayerDecoderEvents {}
 pub enum GenericVar {}
-#[repr(C)]
-pub struct DetectEngineThreadCtx {
-    _unused: [u8; 0],
-}
+pub use suricata_sys::sys::DetectEngineThreadCtx;
```

Comment:
What do you think Jason ? Should I not reexport and use `suricata_sys::sys::DetectEngineThreadCtx` everywhere instead of `core::DetectEngineThreadCtx` ?

Reply:
I think this is fine. Can it be `pub(crate)` though?

Reply:
Good for `pub(crate)`